cpu/atmega_common/periph/gpio.c

```diff
@@ -254,6 +405,66 @@ static inline void irq_handler(uint8_t int_num)
     __exit_isr();
 }
 
+#ifdef PCINT_NUM_BANKS
+/* inline function that is used by the PCINT ISR */
+static inline void pcint_handler(uint8_t bank, volatile uint8_t *mask_reg)
+{
+    __enter_isr();
+    /* Find right item */
+    uint8_t pin_num = 0;
+    uint8_t enabled_pcints = *mask_reg;
+
+    while (enabled_pcints > 0) {
+        /* check if this pin is enabled & has changed */
+        if (enabled_pcints & 0x1) {
+            uint8_t pin_mask = (1 << pin_num);
+            gpio_t pin = pcint_mapping[ bank ][ pin_num ];
+            uint8_t port_value = (_SFR_MEM8(_pin_addr( GPIO_PIN( _port_num(pin), pin_num ))));
+            uint8_t pin_value = (port_value & pin_mask) == 0 ? 0 : 1;
+            uint8_t state = (pcint_state[_port_num(pin)] & pin_mask) == 0 ? 0 : 1;
+            gpio_isr_ctx_pcint_t *conf = &pcint_config[ bank * 8 + pin_num ];
+            if (conf->flank == GPIO_BOTH || ((state & pin_mask) && conf->flank == GPIO_RISING) || (!(state & pin_mask) && conf->flank == GPIO_FALLING)) {
+                pcint_state[_port_num(pin)] ^= (-pin_value ^ pcint_state[_port_num(pin)]) & (1UL << pin_num);
+                /* finally execute callback routine */
+                conf->cb(pcint_config->arg);
+            }
+        }
+        enabled_pcints = enabled_pcints >> 1;
+        pin_num++;
+    }
+
+    __exit_isr();
+}
+#if defined(PCINT0_IDX)
+ISR(PCINT0_vect, ISR_BLOCK)
+{
+    pcint_handler(PCINT0_IDX, &PCMSK0);
+}
+#endif
+
+#if defined(PCINT1_IDX)
+ISR(PCINT1_vect, ISR_BLOCK)
+{
+    pcint_handler(PCINT1_IDX, &PCMSK1);
+}
+#endif  /* PCINT1_vect */
```

Comment:
comment needs to be updated (more occurrences below). Please also add the comment for `PCINT0_IDX` for consistency.

Reply:
Ack.